sunpy/map/mapbase.py

```diff
@@ -2362,7 +2365,20 @@ def draw_contours(self, levels, axes=None, **contour_args):
             # Mask out the data array anywhere the coordinate arrays are not finite
             data = np.ma.array(data, mask=~np.logical_and(np.isfinite(x), np.isfinite(y)))
 
-        cs = axes.contour(x, y, data, levels, **contour_args)
+        if fill:
+            # Ensure we have more than one level if fill is True
+            if len(levels) == 1:
+                max_val = np.nanmax(self.data)
+                # Ensure the existing level is less than max_val
+                if levels[0] < max_val:
```

Comment:
"In `contourf`, at least two levels are necessary to avoid a `ValueError` from `_process_contour_level_args`, as it requires a range to fill between. This contrasts with `contour`, which only draws lines and can operate with a single level."